cmd/backup-manager/app/constants/constants.go

```diff
@@ -17,7 +17,7 @@ import "time"
 
 const (
 	// ResyncDuration is the informer's resync duration
-	ResyncDuration = 5 * time.Second
+	ResyncDuration = 10 * time.Second
```

Comment:
This is 30 seconds in k8s

Reply:
Yes, But in backup、clean and restore job, I want this resync cycle to be faster.